### PR TITLE
Fix logs, so now, old and followed logs has same format without []

### PR DESCRIPTION
**Pull request by LK4D4**
Also, this shows us how often this feature used :)

### CODE REVIEW

**Comment by LK4D4** [comment]
Hah, sorry for that :)

**Comment by SvenDowideit** [comment]
can you please make sure there are no examples in the docs that might also change?

**Comment by LK4D4** [comment]
@SvenDowideit Yup, okay.

**Comment by LK4D4** [comment]
Hah, pretty strange :) seems like Fixed format is not so Fixed

**Comment by LK4D4** [comment]
ping @jfrazelle 
Something bad with drone :/ I can't even rebuild.

File: docs/man/docker-events.1.md [hunk 1]
```diff
@@ -45,23 +45,23 @@ After running docker events a container 786d698004576 is started and stopped
 (The container name has been shortened in the output below):
```
Comment (SvenDowideit) [inline docs/man/docker-events.1.md]: If you're going to change the container ID to long form, can you please remove the above?

Comment (LK4D4) [inline docs/man/docker-events.1.md]: Oh, of course :) It's not me, seems like was changed in code at some time.

**Comment by LK4D4** [comment]
@SvenDowideit Fixed

**Comment by SvenDowideit** [comment]
Docs LGTM - @fredlf @jamtur01

**Comment by jamtur01** [comment]
Docs LGTM

**Comment by fredlf** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**